weather_dl/download_pipeline/util.py

```diff
@@ -72,3 +80,67 @@ def copy(src: str, dst: str) -> None:
     except subprocess.CalledProcessError as e:
         logger.error(f'Failed to copy file {src!r} to {dst!r} due to {e.stderr.decode("utf-8")}')
         raise
+
+
+# TODO(#245): Group with common utilities (duplicated)
```

Comment:
So sorry to ask you to do this, but: Can you duplicate this method in the `loader_pipeline` util.py file until we fix #245?

Reply:
Oh, I see -- this is the copy! 

In that case, can you add this comment to the loader pipeline duplicate :)

Reply:
Done.